### PR TITLE
Translate in Greek language

### PR DESCRIPTION
**Pull request by merianos**
Hello @Alex-D I just translate the Trumbowyg into greek language.

I hope you merge my translation.

Kind regards
Merianos Nikos

### CODE REVIEW

File: src/langs/el.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/* ===========================================================
+ * en.js
+ * English translation for Trumbowyg
```
Comment (Alex-D) [inline src/langs/el.js]: Can you update the header comment with your lang? :)

Comment (merianos) [inline src/langs/el.js]: Yes sure. In a minute I will share the update. Sorry for that :)

**Comment by Alex-D** [comment]
Thanks! :)

**Comment by merianos** [comment]
Thank you too :) :+1:

**Pull request merged**